### PR TITLE
Add support for XQuery file comments and extensions

### LINKED ISSUE #583: support xquery comment syntax
While #177 suggests plugin support for obscure comment styles, the issue seems to have induced no attention and was not yet assigned a milestone. Thus, I suggest to implement the missing syntax the old fashioned way. A pull request with the suggested (minor) code changes will follow and I hope the feature can then soon be introduced.

### PR DESCRIPTION
**Pull request by stexandev**
XQuery only supports multiline comments with a syntax currently
unsupported by the reuse-tool, see
<https://www.w3.org/TR/xquery-31/#comments>.

Close #583

### CODE REVIEW

File: src/reuse/_comment.py [hunk 1]
```diff
@@ -447,6 +448,17 @@ class VimCommentStyle(CommentStyle):
     INDENT_AFTER_SINGLE = " "
 
 
+class XQueryCommentStyle(CommentStyle):
+    """XQuery comment style."""
+
+    _shorthand = "xq"
```
Comment (carmenbianca) [inline src/reuse/_comment.py:454]: I would call this `xquery`

**Review by carmenbianca — CHANGES_REQUESTED** [review]
Thanks for the contribution @stexandev !

A small nitpick.

Also, before this is merged, there needs to be a change log entry, and I believe there's somewhere in the documentation that needs to be updated as well.

File: src/reuse/_comment.py (hunk 1 above)
Comment (stexandev) [reply src/reuse/_comment.py:454]: thank you for your feedback, @carmenbianca; additional commit(s) incoming!

**Comment by stexandev** [comment]
> and I believe there's somewhere in the documentation that needs to be updated as well.

I could not find any place in the documentation where an update would have been necessary.

**Review by mxmehl — APPROVED** [review]
That should be alright, thanks! The first line in the Changelog wouldn't have been necessary though, but we tidy up the changelog before the next release anyway.

**Comment by carmenbianca** [comment]
Merged in #610 

Thank you @stexandev !

**Pull request closed without merging**